### PR TITLE
Thread shrew -- updating our NPM packages

### PR DESCRIPTION
**Pull request by judeallred**
https://www.notion.so/glitch/87bda09f85184c1d81e1659aa62e4392?v=9f7963becc9341ed9af502682c6e9eef&p=d162016dc4694669801639559f840c2c

big items:
 - major verson bump on Babel which has some namespace scope changes
 - several bugfixes from webpack, incl. a memory leak and a caching bug, both of which were probably biting us.  (bundle cache invalidation seems to work well now!)
 - react from 16.4.1 -> 16.5.2  (mostly [bug fixes](https://github.com/facebook/react/blob/master/CHANGELOG.md#1650-september-5-2018) )
- ugpraded node.js from 8.x to 10.x
  -> recap of 10: https://levelup.gitconnected.com/whats-new-in-node-10-ad360ae55ee4
  -> recap of 9: https://nemethgergely.com/what-is-new-in-nodejs-9/

### CODE REVIEW

File: src/.eslintrc.js [hunk 1]
```diff
@@ -35,5 +35,11 @@ module.exports = exports = {
     // Overrides of react/recommended:
     "react/no-unescaped-entities": ["error", {"forbid": [`"`, ">", "}"]}], // permit ' in jsx html,
     "react/prop-types": [OFF], // disabled so we can use composed prop-types
+  },
+  "settings": {
+    "react": {
+      "version": "16.5.2" // Should match package.json
```
Comment (judeallred) [inline src/.eslintrc.js]: specifying the version here is a new config requirement of the react linter

**Comment by GregWeil** [comment]
Looks good, could you also bump up the node version?

**Comment by judeallred** [comment]
will do

**Comment by judeallred** [comment]
upgraded node, too :-)

**Pull request merged**